src/workerd/api/sql.h

```diff
@@ -284,6 +288,11 @@ class SqlStorage::Cursor final: public jsg::Object {
   kj::Maybe<CachedColumnNames> ownCachedColumnNames;
   CachedColumnNames& cachedColumnNames;
 
+  // Invoke when `query.isDone()`, or when we want to prematurely cancel the query. This records
+  // row counters and then sets `stateRef` to `none` to drop the query and return the prepared
```

Comment:
I guess technically it sets `state` to none, not `stateRef`, although they refer to the same thing in most cases.

Reply:
Typo.